src/interfaces/User.ts

```diff
@@ -0,0 +1,20 @@
+export interface UserModel {
+  id: string;
+  login: string;
+  password: string;
+  age: number;
+  isDeleted: boolean;
+}
+
+export interface SuggestRequestQueryModel {
+  loginSubstring?: string;
+  limit?: number;
+}
+
+export type CreateUserModel = Pick<UserModel, 'login' | 'password' | 'age'>;
+export type UpdateUserModel = Pick<UserModel, 'login' | 'password' | 'age' | 'id'>;
```

Comment:
If you update a user, should id change?

Reply:
But it should be provided to update a specific user